src/platform/platform_linux.cpp

```diff
@@ -115,10 +115,16 @@ mp::NetworkInterfaceInfo get_network(const QDir& net_dir)
     static const auto bridge_fname = QStringLiteral("brif");
 
     std::string type, description;
+    std::vector<std::string> links;
     if (auto bridge = "bridge"; net_dir.exists(bridge))
     {
         type = bridge;
         QStringList bridge_members = QDir{net_dir.filePath(bridge_fname)}.entryList(QDir::NoDotAndDotDot | QDir::Dirs);
+
+        links.reserve(bridge_members.size());
+        std::transform(bridge_members.cbegin(), bridge_members.cend(), std::back_inserter(links),
+                       [](const QString& interface) { return interface.toStdString(); });
```

Comment:
This will become unwieldy quickly:
```plain
$ multipass networks
Name             Type      Description
br-enx00133b490  bridge    Network bridge with enx00133b4909cd, tap3b37457c, tap82f64e6f
```

For each instance we'll add a `tap` interface. Maybe we should filter virtual interfaces out? I think for now I'd rather not have any interfaces listed than all of them.

Reply:
Righhht, my bad, forgot about those. Let me fix that.

Reply:
Done.